File: API/ContaCentral/ContaCentral/Domain/Models/Cobranca.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace ContaCentral.Domain.Models
{
    public class Cobranca
    {
        public int Id { get; set; }
        public decimal Valor { get; set; }
        public string Descricao { get; set; }
        public DateTime Data { get; set; }

        public Guid FromUserId { get; set; }
        public ApplicationUser FromUser { get; set; }
        public Guid ToUserId { get; set; }
        public ApplicationUser ToUser { get; set; }

        public int FromCarteiraId { get; set; }
        public Carteira FromCarteira { get; set; }
        public int ToCarteiraId { get; set; }
        public Carteira ToCarteira { get; set; }

        public Boolean CobrancaAtiva { get; set; }
    }
}

File: Website/src/app/domain/models/cobranca.ts
import { Carteira } from "./carteira";
import { User } from "./user";

export class Cobranca{
   id: number;
   valor: number;
   descricao: string;
   data: Date;

   fromUserId: string;
   fromUser: User;

   toUserId: string;
   toUser: User;

   fromCarteiraId: number;
   fromCarteira: Carteira;

   toCarteiraId: number;
   toCarteira: Carteira;

   cobrancaAtiva: Boolean;

   constructor(id: number, Valor: number, Descricao: string, Data: Date, FromUserId: string, FromUser: User, ToUserId: string, ToUser: User, FromCarteiraId: number, FromCarteira: Carteira, ToCarteiraId: number, ToCarteira: Carteira, CobrancaAtiva: Boolean){ 
      this.id = id;
      this.valor = Valor;
      this.descricao = Descricao;
      this.data = Data;

      this.fromUserId = FromUserId;
      this.fromUser = FromUser;

      this.toUserId = ToUserId;
      this.toUser = ToUser;

      this.fromCarteiraId = FromCarteiraId;
      this.fromCarteira = FromCarteira;

      this.toCarteiraId = ToCarteiraId;
      this.toCarteira = ToCarteira;

      this.cobrancaAtiva = CobrancaAtiva;
   }
}